database_scripts/prod_data_investigation.sql

```diff
@@ -0,0 +1,98 @@
+--View a list of submitted ECP claims' eligibility criteria
+select * 
+  from public.early_career_payments_eligibilities
+ where created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14' -- e.g. Datetime format 2021-04-29T14:33:14
+ order by created_at
+
+--View a list of submitted ECP claims
+select * 
+  from public.claims
+ where eligibility_type = 'EarlyCareerPayments::Eligibility'
+ order by created_at;
+
+--Get total number of submitted ECP claims
+select count(*) 
+  from public.claims 
+ where eligibility_type = 'EarlyCareerPayments::Eligibility';
+
+--Get total number of submitted ECP claims
+select count(*), eligibility_type 
+  from public.claims 
+ where created_at BETWEEN '2021-04-01T00:00:00' and '2021-05-31T23:59:59' -- e.g. Datetime format 2021-04-29T14:33:14
+ group by eligibility_type
+ order by count(*);
+
+select * 
+  from public.claims 
+ where created_at BETWEEN '2021-04-01T00:00:00' and '2021-05-31T23:59:59' -- e.g. Datetime format 2021-04-29T14:33:14
+ 
+
+select * 
+  from public.claims 
+ where created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14' -- e.g. Datetime format 2021-04-29T14:33:14
```

Comment:
@and-lucas-kelly the date of 29th October 2021? What does that relate to? I was expecting to see 29th August 2021 if its same as reason for that date in my previous questions.

Reply:
this is purely the investigation script to identify data